### ISSUE TITLE
Module '"flowbite-react"' has no exported member 'Flowbite'.

### ISSUE THREAD

**Issue by lyleunderwood** [themesberg/flowbite-react#1327]
- [x] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [x] I have tested the latest version

## Steps to reproduce

1. Upgrade to 0.7.6
2. Check type errors

## Current behavior

Typescript type exports seem busted in 0.7.6

## Expected behavior

No errors, like in 0.7.5

## Context

I have an up-to-date nextjs / typescript project using `flowbite-react`, and upon upgrading to 0.7.6, all of my imports started having typescript errors. Some examples:

```
src/components/ThemeProvider.tsx:3:26 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("flowbite-react")' call instead.
  To convert this file to an ECMAScript module, add the field `"type": "module"` to '/home/lyle/dev/showlabs/morpheus-frontend/package.json'.

3 import { Flowbite } from 'flowbite-react';
```

```
src/components/ThemeProvider.tsx:3:10 - error TS2305: Module '"flowbite-react"' has no exported member 'Flowbite'.

3 import { Flowbite } from 'flowbite-react';
```

```
src/components/SearchInput.tsx:4:10 - error TS2305: Module '"flowbite-react"' has no exported member 'TextInput'.

4 import { TextInput } from 'flowbite-react';
```

I'm pretty sure that every single import from `flowbite-react` errors. Rolling back to 0.7.5 resolves the issue. I assume this is related to the monorepo transition in 0.7.6.

**Comment by SutuSebastian** [comment]
I have an idea what could be wrong. I'll be back with more info.

**Comment by rluders** [comment]
Probably related/connected to #1329 1329

**Comment by SutuSebastian** [comment]
Check `0.7.7`, hope it fixed everything related to imports/build/install.

**Comment by lyleunderwood** [comment]
I had some auto-import references like:

```
import { Button } from 'flowbite-react/lib/esm/components/Button';
```

which no longer resolved in `0.7.8`, just had to update them and now everything seems to work fine :+1:
